packages/adp-tooling/test/unit/writer/__snapshots__/index.test.ts.snap

```diff
@@ -17,14 +17,15 @@ Object {
     \\"devDependencies\\": {
         \\"@sap/ux-ui5-tooling\\": \\"1\\",
         \\"@ui5/task-adaptation\\": \\"^1.3.0\\",
-        \\"@ui5/cli\\": \\"^3.9.2\\"
+        \\"@ui5/cli\\": \\"^3.9.2\\",
+        \\"rimraf\\": \\"^5.0.5\\"
     },
     \\"scripts\\": {
         \\"build\\": \\"ui5 build --exclude-task generateFlexChangesBundle generateComponentPreload minify --clean-dest\\",
         \\"start\\": \\"fiori run --open /test/flp.html#app-preview\\",
         \\"start-editor\\": \\"fiori run --open /test/adaptation-editor.html\\",
-        \\"deploy\\": \\"ui5 build --config ui5-deploy.yaml --exclude-task deploy-to-abap generateFlexChangesBundle generateComponentPreload --clean-dest && fiori deploy --config ui5-deploy.yaml\\",
-        \\"undeploy\\": \\"fiori undeploy --config ui5-deploy.yaml --lrep \\\\\\"apps/the.original.app/appVariants/my.test.app/\\\\\\"\\",
+        \\"deploy\\": \\"npm run build && fiori deploy --config ui5-deploy.yaml && rimraf archive.zip\\",
```

Comment:
`.gitignore` should also exclude this from source control which I think from looking at the snapshots, is appended i.e. `*.zip`